common/aepEdgeClient.js

```diff
@@ -191,6 +191,53 @@ function createAepEdgeClient(
       });
   }
 
+  function collect(requestBody, requestHeaders = {}) {
+    const requestId = uuidv4();
+
+    let domain = edgeDomain;
+    let region = aepEdgeCluster;
+
+    if (edgeDomain === "server.adobedc.net") {
```

Comment:
what is the difference between `server.adobedc.net` and `edge.adobedc.net`?

Reply:
The `server.adobedc.net` is the domain that is available for the Server API, which enables authenticated requests, more details: https://experienceleague.adobe.com/en/docs/experience-platform/edge-network-server-api/overview.

We think this should be the default choice for server side integrations. This is why I added also the IMS authentication: https://github.com/adobe/alloy-samples/pull/20/files#diff-080ae08ca8872743e902274ef405ef367560d8ae70bd0cfd60709e152031613bR27, which is needed when using this endpoint.